frontend/src/lib/Main.svelte

```diff
@@ -1,11 +1,20 @@
 <script>
+  import { onMount } from 'svelte'
+  import { push, location } from 'svelte-spa-router'
   import { Button } from 'carbon-components-svelte'
   import { SkeletonPlaceholder } from 'carbon-components-svelte'
   import { currentWord, randomize } from './stores'
+  import languages from '../languages/languages.json'
 
   import 'carbon-components-svelte/css/g10.css'
   import 'carbon-components-svelte/css/all.css'
 
+  onMount(() => {
+    if ($location === '/') {
+      push(`/${languages[0].code}`)
```

Comment:
This does not match what the task says it should be.

Reply:
It should also not use `push` because that leaves people clicking Back and just getting redirected again.